component/azstorage/datalake.go

```diff
@@ -348,7 +361,9 @@ func (dl *Datalake) GetAttr(name string) (attr *internal.ObjAttr, err error) {
 	log.Trace("Datalake::GetAttr : name %s", name)
 
 	fileClient := dl.Filesystem.NewFileClient(filepath.Join(dl.Config.prefixPath, name))
-	prop, err := fileClient.GetProperties(context.Background(), nil)
+	prop, err := fileClient.GetProperties(context.Background(), &file.GetPropertiesOptions{
```

Comment:
Other than rename and getProperties we do not need the CPK in any other call. What about Set/Get AccessControl apis.

Reply:
There is no field for CPK in the options for those apis